order-command-ms/chart/ordercommandms/values.yaml

```diff
@@ -41,10 +41,10 @@ kafka:
   topicsConfigMap: kafka-topics
 eventstreams:
   enabled: true
-  apikeyConfigMap: eventstreams-apikey
+  esCredSecret: eventstreams-cred
   truststoreRequired: false
   truststorePath: /config/resources/security/es-ssl
-  truststoreFile: es-cert.jks
-  truststoreSecret: es-truststore-jks
-  truststorePassword: password
+  truststoreFile: es-cert.p12
```

Comment:
It's much nicer to be passing in the certificate rather than a truststore.  Would it be worth renaming 'truststore' to 'certificate'?  (does not necessarily need to be part of this PR)

Reply:
@jesusmah will circle back on this via https://github.com/ibm-cloud-architecture/refarch-kc-gitops/pull/16#issuecomment-671953346 after initial integration.